sql/core/src/test/scala/org/apache/spark/sql/connector/TestV2SessionCatalogBase.scala

```diff
@@ -64,6 +64,15 @@ private[connector] trait TestV2SessionCatalogBase[T <: Table] extends Delegating
     }
   }
 
+  override def createTable(
+      ident: Identifier,
+      columns: Array[Column],
+      partitions: Array[Transform],
+      properties: java.util.Map[String, String]): Table = {
+    createTable(ident, CatalogV2Util.v2ColumnsToStructType(columns), partitions, properties)
+  }
+
+  // TODO: remove it when no tests calling this deprecated method.
```

Comment:
Is there a follow-up ticket for this?

Reply:
I haven't created it. This is a test-only change so the priority is low.